cmd/libsnap-confine-private/snap-dir.h

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright (C) 2024 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+#ifndef SNAP_CONFINE_SNAP_DIR_H
+#define SNAP_CONFINE_SNAP_DIR_H
+
+#include "error.h"
+
+/**
+ * Preferred location of the mount tree where snaps are visible on the system or
+ * the location of the symbolic link to the fallback location.
+ **/
+#define SC_PREFERRED_SNAP_MOUNT_DIR "/snap"
```

Comment:
```suggestion
#define SC_CANONICAL_SNAP_MOUNT_DIR "/snap"
```
:rofl:, feel free to ignore

Reply:
OOOOOH LOL

Reply:
Done :)